plugins/autocomplete/plugin.js

```diff
@@ -150,6 +150,9 @@
 	 * {@link CKEDITOR.plugins.autocomplete.model.item} interface.
 	 */
 	function Autocomplete( editor, textTestCallback, dataCallback ) {
+		var commitKeystroke = CKEDITOR.config.autocomplete_commitKeystroke;
+		commitKeystroke = CKEDITOR.tools.array.isArray( commitKeystroke ) ? commitKeystroke.slice() : commitKeystroke;
```

Comment:
This is a stray line. I see why you used `commitKeystroke` - simply because `CKEDITOR.config.autocomplete_commitKeystroke` is long and you want to make an alias - good decision.

However the above line is stray and it could be inlined to `this.commitKeystroke` initialization, like so:

```js
this.commitKeystroke = CKEDITOR.tools.array.isArray( configKeystroke ) ? configKeystroke.slice() : configKeystroke;
```

And with this change I find `configKeystroke` name to be more appropriate.